students/shibin_mathew/lesson6/mailroom.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python3
+import sys
+from collections import defaultdict
+
+
+def print_donor_names(donors2):
+    list_test = []
+    for names in donors2:
+        print(names)
+        list_test.append(names)  # Debug - for unit testing
+    return list_test
+
+
+def send_thank_you(donors2):
+    #  Function to Send a Thank You Email
+    program_quit = False
+    while not program_quit:
+        name = input("Please input your full name\n")
+        if name == "list":
+            print("list of names")
+            print_donor_names(donors2)
+        elif name == 'quit' or name == 'Quit':
+            break
+        else:
+            break
+    while True:
+        donation = input("Please enter a donation amount\n")
+        if donation == 'quit' or donation == 'Quit':
+            return
+        else:
+            try:
+                donation = float(donation)
+                donors2[name].append(donation)
+                break  # break out of the while loop
+            except ValueError:
+                print("please input a valid amount")
+                continue
+    return None
```

Comment:
Python functions return `None` by default, so there's no need to include this statement here